SYCL/Plugin/sycl-ls.cpp

```diff
@@ -11,4 +11,4 @@
 //===----------------------------------------------------------------------===//
 // The test crashed on CUDA CI machines with the latest OpenCL GPU RT
 // (21.19.19792).
-// UNSUPPORTED: cuda
+// UNSUPPORTED: cuda || rocm
```

Comment:
Why is generic "sycl-ls" not working? Can you at least add a comment?

Reply:
It appears to have been a mistake, the test works as intended with ROCm. I am removing the change.